src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -390,6 +399,33 @@ void ViewSceneDialog::assignBackgroundColor()
 }
 
 //------------------------------------------------------------------------------
+void ViewSceneDialog::handleUnselectedItem(const QString& name)
+{
+  unselectedObjectNames_.push_back(name.toStdString());
+  newGeometryValue();
```

Comment:
Heh, the combo box we're using doesn't allow a ton of clicks in a row, preventing this goofy behavior. We can examine the performance later